shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -52,13 +52,14 @@
     private BinaryMessenger mMessenger;
 
     // The accessibility bridge to which accessibility events form the platform views will be dispatched.
-    private AccessibilityBridge accessibilityBridge;
+    private final CurrentAccessibilityBridge mCurrentAccessibilityBridge;
```

Comment:
nit: Please rename to just `currentAccessibilityBridge` for this and the related variables.

https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names

Reply:
Our codebase right now is in this weird state were part of the newer files are following the Google Style guide and files from up to around 8 months ago are following something similar to the Android style guide.

We should migrate to a single style, but I think it should be done in dedicated PRs, and I wouldn't want to introduce mixed style to this file in this PR. I think we should follow up with a separate PR to rename all the fields here.

Reply:
SGTM.